### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by kianmeng**
Found via `codespell -S *.html,.git`

### CODE REVIEW

File: animated_drawings/config.py [hunk 1]
```diff
@@ -459,7 +459,7 @@ def __init__(self, retarget_cfg_fn: str) -> None:  # noqa: C901
             self.bvh_projection_bodypart_groups = retarget_cfg['bvh_projection_bodypart_groups']
 
             for group in self.bvh_projection_bodypart_groups:
-                assert group['method'] in ['pca', 'saggital', 'frontal'], 'group method must be "pca", "saggital", or "frontal"'
+                assert group['method'] in ['pca', 'sagittal', 'frontal'], 'group method must be "pca", "sagittal", or "frontal"'
```
Comment (hjessmith) [inline animated_drawings/config.py:462]: This misspelling needs to be corrected in the example retarget config files and readme docs at the same time as it is corrected here.

File: animated_drawings/config.py [hunk 2]
```diff
@@ -410,7 +410,7 @@ def __init__(self, motion_cfg_fn: str) -> None:  # noqa: C901
             assert False, msg
 
     def validate_bvh(self, bvh_joint_names: List[str]) -> None:
-        """ Performs all the validation steps that depend upon knowing the BVH joint names. This should be called once the BVH had been leaded """
+        """ Performs all the validation steps that depend upon knowing the BVH joint names. This should be called once the BVH had been led """
```
Comment (hjessmith) [inline animated_drawings/config.py:413]: Should be 'loaded'

**Review by Aayog — APPROVED** [review]

**Review by hjessmith — APPROVED** [review]
LGTM.

Github CI tests are failing due to reasons unrelated to contents of this PR. I've confirmed locally that all tests pass and it's okay to merge.

**Comment by kianmeng** [comment]
:tada: :tada: :tada: :tada: :tada:

**Pull request merged**